src/get_env.c

```diff
@@ -40,28 +33,12 @@ rcutils_get_env(const char * env_name, const char ** env_value)
   if (NULL == env_value) {
     return "argument env_value is null";
   }
-  *env_value = NULL;
-#ifdef _WIN32
-  size_t required_size;
-  errno_t ret = getenv_s(&required_size, __env_buffer, sizeof(__env_buffer), env_name);
-  switch (ret) {
-    case 0:
-      __env_buffer[WINDOWS_ENV_BUFFER_SIZE - 1] = '\0';
-      *env_value = __env_buffer;
-      break;
-    case EINVAL:
-      return "invalid arguments when reading environment variable";
-    case ERANGE:
-      return "insufficient buffer size to read environment variable";
-    default:
-      return "unknown error code reading environment variable";
-  }
-#else
   *env_value = getenv(env_name);
+  // Note: getenv is deprecated; consider using getenv_s instead
```

Comment:
This note should be above the line it talks about, and should be something more like:

```suggestion
  // TODO(Suyash458): getenv is deprecated on Windows; consider using getenv_s instead
```

Reply:
Sure, will update this and create a new issue

Reply:
@clalancette do we want to close [this](https://github.com/ros2/rcutils/issues/30) and create a new issue or just update it with these discussions?

Reply:
Let's close #30 (which will happen just by merging this PR) and open a new one.